syncd/NotificationProcessor.cpp

```diff
@@ -397,7 +397,20 @@ void NotificationProcessor::process_on_port_state_change(
          * switch vid.
          */
 
-        oper_stat->port_id = m_translator->translateRidToVid(oper_stat->port_id, SAI_NULL_OBJECT_ID);
+        SWSS_LOG_DEBUG("Port RID %s state change notification", 
+                sai_serialize_object_id(oper_stat->port_id).c_str());
+
+        if (false == m_translator->tryTranslateRidToVid(oper_stat->port_id, oper_stat->port_id))
+        {
+            oper_stat->port_id = SAI_NULL_OBJECT_ID;
```

Comment:
does it make sense to send notification with null port ?
log warning message here, that RID was not found and you skip this notification, and just add "continue"

Reply:
Adding WARN log is fine. If not sending  NULL object then  "oper_stat->port_id" holds invalid RID otherwise need to adjust  "data" and "count" variable data (delete NULL entries). Need to build fresh entries. Will do it that in next check-in, let report NULL object for now. Agree?

Reply:
ok